src/sentry/tasks/store.py

```diff
@@ -621,7 +628,9 @@ def save_event_transaction(
     project_id: int | None = None,
     **kwargs: Any,
 ) -> None:
-    _do_save_event(cache_key, data, start_time, event_id, project_id, **kwargs)
+    _do_save_event(
+        ConsumerType.Transactions, cache_key, data, start_time, event_id, project_id, **kwargs
```

Comment:
There is also a call to `_do_save_event` on line 614. That call is not using named parameters and won't behave correctly with the change in `_do_save_event`'s signature.